src/test/java/template/neodymium/tests/smoke/HomePageTest.java

```diff
@@ -4,20 +4,17 @@
 
 import com.xceptance.neodymium.util.Neodymium;
 
-import template.flows.OpenHomePageFlow;
+import template.flows.OpenPageFlows;
 import template.neodymium.tests.AbstractTest;
-import template.pageobjects.pages.HomePage;
 
 public class HomePageTest extends AbstractTest
 {
     @Test
     public void testVisitingHomepage()
     {
         // Goto the home page
```

Comment:
extend the comment: 
`// Goto the home page and perform a short validation that we are on the correct page`

Reply:
Done